components/browser/toolbar/src/main/res/values/strings.xml

```diff
@@ -6,6 +6,10 @@
     <!-- Content description (not visible, for screen readers etc.): Description for the overflow menu button in the browser toolbar. -->
     <string name="mozac_browser_toolbar_menu_button">Menu</string>
     <string name="mozac_clear_button_description">Clear</string>
+    <!-- Content description: For the tracking protection toolbar icon, it is set when the site has tracking protection enabled, but none trackers have been blocked or detected. -->
+    <string name="mozac_browser_toolbar_content_description_tracking_protection_on">Tracking Protection is on</string>
```

Comment:
Where did these content descriptions come from?

Reply:
I added them. As every icon means a tracking protection state, we want to communicate that to the accessibility tools like screen readers. I'm not quite sure if they are completely meaningful for these I issued https://github.com/mozilla-mobile/android-components/issues/3925 for investigating about it.

Reply:
CC @mheubusch